jupyter-on-gke/variables.tf

```diff
@@ -15,13 +15,12 @@
 variable "namespace" {
   type        = string
   description = "Kubernetes namespace where resources are deployed"
-  default     = "<your user name>"
+  default     = "umesh"
```

Comment:
revert this?

Reply:
this wont be used as user inputs.. its just a placeholder.

will anyway update in new PR